### ISSUE TITLE
Add a filter to show_linked_elements

### ISSUE THREAD

**Issue by mbootsman** [inpsyde/multilingual-press#241]
Can a filter be added to `show_linked_elements` so that it's easy to change,add or remove something from the linked elements output?

**Comment by tfrommen** [comment]
You mean like proposed in #228? ;)

We have this planned, yes, but not as part of a patch release. And since we are already working on the next major, this will be included in MultilingualPress 3.0.0.

The current workaround would be to use a custom function, like so:

```php
function my_mlp_linked_elements( $args ) {

    ob_start();
    mlp_show_linked_elements( $args );
    echo apply_filters( 'my_filter', ob_get_clean() );
}
```

Instead of providing your own filter, you can of course just go ahead and manipulate the output like you want it to be - if this is always the same.

Hope this helps!
Thorsten

**Comment by mbootsman** [comment]
That is exactly what I mean. Thanks

*closing*
